### ISSUE TITLE
[Probably a bug]Seer and completing farm levels

### ISSUE THREAD

**Issue by lstawowy** [Heishe/idleraiders-second-run#594]
I cannot guarantee that but i think there is a bug that when earlier i wasnt able to complete farm map lvl 120 or 121 but after defeating Seer i had it completed with once 30/10 kills and second time it was 18/9 kills... I might miss something but for me that was quite weird...

**Comment by Meissner-Effect** [comment]
Yes, most of the raid bosses seem to count as current farm map enemies.

**Comment by DaemoniaV** [comment]
You can kill first 2 boss of group of 3 then cancel raid and repeat. So it can abuse(

**Comment by Meissner-Effect** [comment]
Good point - the 2/3 boss kill for map progression could make it possible for anyone to climb the farm map levels for the one-time rewards even if their raiders are no where near capable of handling those levels. Definitely a bug

**Comment by TravisGesslein** [comment]
Thanks for the bug report, I've been told about this from multiple sources and want to fix it ASAP. My work at the moment is split between bug fixing and preparing 1.1 so we can roll out a beta, so apologies if fairly urgent issues like these go unfixed for longer than usual.
